stores/use-windows-store.ts

```diff
@@ -249,6 +249,20 @@ export const useWindowsStore = defineStore("windows", () => {
 			root: rootElement,
 		});
 
+		if (hasContentId(params.params)) {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-call
+			winbox.addControl({
+				index: 0,
+				class: "wb-tei",
+				click: function () {
+					const env = useRuntimeConfig();
+					if (env.public.TEI_BASEURL) {
+						window.open(`${env.public.TEI_BASEURL}&id=${params.params.id}`, "_blank");
```

Comment:
We didn't talk about how this should work. It is not so easy as this should point the user to the vicav-content repository and the correct file there.  
id here refers to an @xml:id in the XML which does not correspond to the file name. 
The differences is mostly in spelling but not exclusively.  
The text would need to transport its filename in the database, that would be something that is related to the content repository.

Reply:
simple example: We need to point [vicavMission](https://github.com/acdh-oeaw/vicav-content/blob/master/vicav_texts/vicav_mission.xml#L37) to https://github.com/acdh-oeaw/vicav-content/blob/master/vicav_texts/vicav_mission.xml but also
[vicavOverview_corpora_historical_varieties](https://github.com/acdh-oeaw/vicav-content/blob/master/vicav_texts/vicav_overview_corpora_historical_varieties.xml#L22) to https://github.com/acdh-oeaw/vicav-content/blob/master/vicav_texts/vicav_overview_corpora_historical_varieties.xml .  
I am sure there are further exceptions.

Maybe it is an option to have a 302 kind of trampoline service in the backend that takes an xml:id and responds with the GitHub URL this should point to.

Reply:
@simar0at If the xml:id cannot be automatically transformed to the filename, IMHO the simplest would be to send it from the backend along with the id. Then we would change the logic so that instead of the content id (params.id) we would check for the filename (e. g. params.teiSource), and display the TEI button whenever we have it.

Reply:
I will change the backend to report these `teiSource` properties.

Reply:
Changed the criteria for showing the TEI button to the presence of the teiSource parameter, see https://github.com/acdh-oeaw/vicav-vue3/pull/99/commits/1c85e8527d31a29b5edd9bb210ae68ba20317057

Reply:
@simar0at please press the "Resolve" button below if you agree with the outcome